app/controllers/admin/popular_articles_controller.rb

```diff
@@ -0,0 +1,18 @@
+class Admin::PopularArticlesController < Admin::BaseController
+
+  def new
+    @popular_article = PopularArticle.last || PopularArticle.new
```

Comment:
Might be nice to expose something like `PopularArticle.last_or_initialize` that wraps that behaviour.